win32ss/user/ntuser/winsta.h

```diff
@@ -40,8 +38,21 @@ typedef struct _WINSTATION_OBJECT
     LUID           luidUser;
     PVOID          psidUser;
 
+    struct _DESKTOP* ActiveDesktop;
+    HANDLE         ShellWindow;
+    HANDLE         ShellListView;
 } WINSTATION_OBJECT, *PWINSTATION_OBJECT;
 
+#ifndef _WIN64
```

Comment:
As said in the JIRA report, this structure is internal to win32k only so it's not mandatory to have the very same layout (especially when fields are different due to fundamental differences in implementations).